CONTRIBUTING.md

```diff
@@ -0,0 +1,152 @@
+# Contributing
+
+## General Workflow
+
+1. Fork the repo
+1. Cut a namespaced feature branch from development
+1. Make commits to your feature branch. Prefix each commit like so:
+1. When you've finished with your fix or feature, Rebase upstream changes into your branch. submit a pull request directly to master. Include a description of your changes.
+1. Your pull request will be reviewed by another maintainer. The point of code reviews is to help keep the codebase clean and of high quality and, equally as important, to help you grow as a programmer. If your code reviewer requests you make a change you don't understand, ask them why.
+1. Fix any issues raised by your code reviewer, and push your fixes as a single new commit.
+1. Once the pull request has been reviewed, it will be merged by another member of the team. Do not merge your own commits.
+
+## Detailed Workflow
+
+### Fork the repo
```

Comment:
Did we want to use the forking method? I prefer the standing cloning method since we all part of the same org. Comments? @enriq018 @tiffaphm @andino93

Reply:
We could certainly merge this and then adjust this doc later if you guys prefer.

Reply:
The forking method is for anyone who isn't in our org who wants to contribute. We would still follow our current workflow. Since we have a public repo, I figured having the contributing pages written with that in mind would make more sense. I'm open to changing it though if its unclear!

Reply:
No worries. that's fine. Just as long as we are on the same page about how we are working as an org that's fine.